src/EnergyPlus/AirflowNetworkBalanceManager.hh

```diff
@@ -232,6 +233,63 @@ namespace AirflowNetworkBalanceManager {
 	Real64
 	GetZoneInfilAirChangeRate( int const ZoneNum ); // hybrid ventilation system controlled zone number
 
+	// derived class or struct
+	struct OccupantVentilationControlProp {
+
```

Comment:
Based on a conference call on 2/20/15, int const will be used, instead of enum class, dur to possible conflict of variable types between enum and int.